config/routes.rb

```diff
@@ -39,8 +39,23 @@
 
   ### Statutory Instruments ###
   get '/statutory-instruments', to: 'statutory_instruments#index', as: :statutory_instruments
+
   get '/statutory-instruments/:statutory_instrument_id', to: 'statutory_instruments#show', as: :statutory_instrument
```

Comment:
With the below scope, this is no longer needed

Reply:
🎉